Neos.Ui/core/src/application/LinkTypes/PhoneNumber/PhoneNumber.tsx

```diff
@@ -61,7 +64,11 @@ export const PhoneNumber = makeLinkType<PhoneNumberLinkModel, PhoneNumberLinkOpt
     },
 
     Editor: ({model, options}: {model: Nullable<PhoneNumberLinkModel>, options: OptionalDeep<PhoneNumberLinkOptions>}) => {
+        const [codeArea, setCodeArea] = useState<string>(model?.countryCallingCode || (options?.defaultCountry ? `+${getCountryCallingCode(options?.defaultCountry).toString()}` : `+${getCountryCallingCode(getCountries()[0]).toString()}`));
```

Comment:
I would extract the conditional logic for a default country calling code into a variable and use the variable in the `useState` initialisation. This would improve the readability.

Reply:
I defined a new variable defaultCountryCallingCode.